pillar.example

```diff
@@ -1,6 +1,31 @@
 mongodb:
   lookup:
-    pkg: mymongodb
-    conf_file: /usr/local/etc/mongodb/mongodb.conf
-  config:
-    key: othervalue
+    install:
```

Comment:
Yes, I kept it there to make it easier for someone to copy the pillar example and start filling in overrides. Let me know if this seems like it would be weird or confusing to users.